diff --git a/src/app-layer-detect-proto.c b/src/app-layer-detect-proto.c
index f9a5451..ad777e7 100755
--- a/src/app-layer-detect-proto.c
+++ b/src/app-layer-detect-proto.c
@@ -913,6 +913,8 @@ static void AppLayerProtoDetectPrintProbingParsers(AppLayerProtoDetectProbingPar
                         printf("            alproto: ALPROTO_MQTT\n");
                     else if (pp_pe->alproto == ALPROTO_PGSQL)
                         printf("            alproto: ALPROTO_PGSQL\n");
+                    else if (pp_pe->alproto == ALPROTO_S7COMM)
+                        printf("            alproto: ALPROTO_S7COMM\n");
                     else if (pp_pe->alproto == ALPROTO_TELNET)
                         printf("            alproto: ALPROTO_TELNET\n");
                     else if (pp_pe->alproto == ALPROTO_TEMPLATE)
@@ -998,6 +1000,8 @@ static void AppLayerProtoDetectPrintProbingParsers(AppLayerProtoDetectProbingPar
                     printf("            alproto: ALPROTO_MQTT\n");
                 else if (pp_pe->alproto == ALPROTO_PGSQL)
                     printf("            alproto: ALPROTO_PGSQL\n");
+                else if (pp_pe->alproto == ALPROTO_S7COMM)
+                    printf("            alproto: ALPROTO_S7COMM\n");
                 else if (pp_pe->alproto == ALPROTO_TELNET)
                     printf("            alproto: ALPROTO_TELNET\n");
                 else if (pp_pe->alproto == ALPROTO_TEMPLATE)