diff --git a/src/util-profiling.c b/src/util-profiling.c
index aadbe66..d0ea098 100755
--- a/src/util-profiling.c
+++ b/src/util-profiling.c
@@ -1275,6 +1275,7 @@ const char * PacketProfileLoggertIdToString(LoggerId id)
         CASE_CODE (LOGGER_TCP_DATA);
         CASE_CODE (LOGGER_JSON_FLOW);
         CASE_CODE (LOGGER_JSON_NETFLOW);
+        CASE_CODE (LOGGER_JSON_S7COMM);
         CASE_CODE (LOGGER_STATS);
         CASE_CODE (LOGGER_JSON_STATS);
         CASE_CODE (LOGGER_PCAP);